src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -515,31 +515,6 @@ public void canGetRenewedLoan() {
       withinSecondsAfter(Seconds.seconds(10), approximateRenewalDate.plusWeeks(3)));
   }
 
-  @Test
-  public void cannotRenewWhenLoanPolicyDoesNotExist() {
```

Comment:
It is still possible for the rules to refer to a non-existent loan policy when a request is created. Why has this test been removed?

Reply:
I returned it back and little bit updated

Reply:
Thanks